sdk/communication/communication-phone-numbers/swagger/README.md

```diff
@@ -7,12 +7,12 @@
 ```yaml
 package-name: "@azure/communication-phone-numbers"
 description: Phone number configuration client
-package-version: 1.2.0-beta.3
+package-version: 1.2.0-beta.4
 generate-metadata: false
 license-header: MICROSOFT_MIT_NO_VERSION
 output-folder: ../src/generated
-tag: package-phonenumber-2022-01-11-preview2
-require: https://raw.githubusercontent.com/Azure/azure-rest-api-specs/02fce64677f78021ba70d69bb8bd0d916d653927/specification/communication/data-plane/PhoneNumbers/readme.md
+tag: package-phonenumber-2022-06-01-preview
+require: https://github.com/Azure/azure-rest-api-specs/blob/6de2e5bf9286b2dddea9a372c78cc0de214897dd/specification/communication/data-plane/PhoneNumbers/readme.md
```

Comment:
Any particular reason to not use the raw content?

Reply:
Yes, it's because the swagger hasn't been merged yet. We can rectify this, once it's deployed (should be soon).